common/utils.js

```diff
@@ -785,6 +785,39 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function estimateITSFee(chain, destinationChain, env, eventType, gasValue, gmpAxelarscanApi) {
+    if (!gmpAxelarscanApi || env === 'devnet-amplifier') {
+        printInfo('Using default ITS fee: 0');
+        return 0;
+    }
+
+    if (isValidNumber(gasValue)) {
+        return scaleGasValue(chain, gasValue);
+    }
+
+    const url = `${gmpAxelarscanApi}/estimateITSFee`;
+
+    const payload = {
+        sourceChain: chain.axelarId,
+        destinationChain: destinationChain,
+        event: eventType,
+    };
+
+    const res = await httpPost(url, payload);
+    if (res.error) {
+        throw new Error(`Error querying gas amount: ${res.error}`);
+    }
+    return res;
+}
+
+function scaleGasValue(chain, gasValue) {
+    if (typeof chain.gasScalingFactor === 'number') {
```

Comment:
Where is gas scaling factor defined? Don't chains already have a token decimals in their config that can be reused

Reply:
it's defined in the chains config file. Only hedera related assets makes use of this field. Noting that this fn was previously defined I had just moved it here to make sure there is no circular dependency between this function and the newly added `estimateItsFee`